.github/workflows/security-scans.yml

```diff
@@ -147,12 +147,15 @@ jobs:
           cd operator/config/default
           sed -i 's/\- ..\/crd//' kustomization.yaml && kustomize build ./ > /tmp/operator-manifest.yaml
           cd ../crd && kustomize build ./ > /tmp/crds.yaml
+          cd ../../../metrics-operator/config/default
+          sed -i 's/\- ..\/crd//' kustomization.yaml && kustomize build ./ > /tmp/metrics-operator-manifest.yaml
+          cd ../crd && kustomize build ./ > /tmp/crds.yaml
```

Comment:
This will overwrite whatever was in `/tmp/crds.yaml` from the previous `cat`

Reply:
fixed, thx!